src/test/kotlin/graphql/kickstart/tools/MethodFieldResolverDataFetcherTest.kt

```diff
@@ -176,20 +177,19 @@ class MethodFieldResolverDataFetcherTest {
 
     @Test
     fun `data fetcher passes environment if method has extra argument even if context is specified`() {
-        val options = SchemaParserOptions.newOptions().contextClass(ContextClass::class).build()
-        val resolver = createFetcher("active", options = options, resolver = object : GraphQLResolver<DataClass> {
+        val context = GraphQLContext.newContext().build()
+        val resolver = createFetcher("active", resolver = object : GraphQLResolver<DataClass> {
             fun isActive(dataClass: DataClass, env: DataFetchingEnvironment): Boolean = env is DataFetchingEnvironment
         })
 
-        assertEquals(resolver.get(createEnvironment(DataClass(), context = ContextClass())), true)
+        assertEquals(resolver.get(createEnvironment(DataClass(), context = context)), true)
     }
 
     @Test
     fun `data fetcher passes context if method has extra argument and context is specified`() {
-        val context = ContextClass()
-        val options = SchemaParserOptions.newOptions().contextClass(ContextClass::class).build()
-        val resolver = createFetcher("active", options = options, resolver = object : GraphQLResolver<DataClass> {
-            fun isActive(dataClass: DataClass, ctx: ContextClass): Boolean {
+        val context = GraphQLContext.newContext().build()
```

Comment:
This test needs to be changed back to test out that custom context classes still work. Or maybe split it in two tests if we want both.

Reply:
Added a new test to reflect that.